modules/ROOT/pages/policy-mule3-add-headers-policy.adoc

```diff
@@ -0,0 +1,112 @@
+// * Ensure that you are licensed to use API Manager functionality (Mule 3.8.x runtime) or API Gateway runtime 2.x.
+
+// == Downloading the Header Injection Policy
+
+// Currently, the Header Injection policy is available on Anypoint Exchange. The follow section describes the steps to download, set up, and apply these policies.
+
+// To download the header policies:
+
+// . Login to Anypoint Platform and go to Exchange.
+// . Search for and download the policies, or use these URLs:
+// +
+// * Add Request Headers Policy
+// +
+// `+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Request%20Headers%20Policy+`
+// +
+// //* Remove Request Headers Policy
+// //+
+// //`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Request%20Headers%20Policy+`
+// //+
+// * Add Response Headers Policy
+// +
+// `+https://www.anypoint.mulesoft.com/exchange/?search=Add%20Response%20Headers%20Policy+`
+// //+
+// //* Remove Response Headers Policy
+// //+
+// //`+https://www.anypoint.mulesoft.com/exchange/?search=Remove%20Response%20Headers%20Policy+`
+// +
+// . Unzip each policy archive.
+
+// An XML and YAML file appears in each of the folders created by unzipping the archives. For example, the add-request-header-policy folder contains the following files:
+
+// * add-request.xml
+// * add-request.yaml
+
+// You use these files to set up the policy as described later.
+
+
+// == Setting Up and Applying Header Policies
+
+// If you have configured custom policies in Anypoint Platform, you already know how to configure the Header Injection policy. The following section describes how to set up and apply the policies in Anypoint Platform.
+
+// To configure the Header Injection Policy:
+
+// . Navigate to the main API Administration page.
+// +
+// . From the left navigation, select Custom Policies.
+// +
+// The Custom policies page appears listing any available custom policies for the API.
+// +
+// . Click *Add Custom Policy*.
+// . In *Mule Version*, select the version of the Mule Runtime for which you want to configure the policy.
+// . In *Name*, enter a name for the policy, for example `add request header policy`.
+// . In *Policy Definition*, click *Choose File* to locate the YAML file that you downloaded (`add-request.yaml`).
+// . In Policy Configuration, choose the XML file that you downloaded, for example, `add-request.xml`.
+// . Click *Add*.
+// +
+// image::gw-add-custom-pol.png[]
+// +
+// The policy, add request header policy in this example, appears on the Custom Policies page.
+// +
+// image::gw-custom-pol-list.png[]
+
+== Applying the Header Injection Policy
+
+Perform the following steps to apply the policy:
+
+. From your left navigation, select *API Administration*.
+. Select the version to which you want to apply the policy.
+. From your left navigation, select *Policies*.
++
+The policies details page appears. If the header was already provided by the client or another policy, the value is overwritten with your header.
```

Comment:
no

Reply:
I assume you meant that the value is not overwritten by the header. I removed the sentence. A bit more detail about "no" for what would have helped.